### ISSUE TITLE
[Feature Request] Ability to disable multiconnect in-game

### ISSUE THREAD

**Issue by Soham52** [Earthcomputer/multiconnect#226]
There should be a in-game setting in mod-menu which can disable multiconnect because it can be advantageous in some situations. For example when a player wants to join a modded server of the current version which would not be possible with multiconnect enabled

**Comment by Earthcomputer** [comment]
You need a different instance of Minecraft anyway in that case.

**Comment by Soham52** [comment]
No, when i am playing on 1.17.1 and enter my server it always gives me an error and does not let me in, but i and my other friends can always enter it with a vanilla client

**Comment by Earthcomputer** [comment]
Please show the error

**Comment by Soham52** [comment]
https://github.com/Earthcomputer/multiconnect/issues/216#issue-963134563
This exact error

**Comment by Soham52** [comment]
So will this be added?

**Comment by Earthcomputer** [comment]
I'm planning a refactor of the way multiconnect handles registries, it's possible it may fix this issue, then I wouldn't have to think about disabling multiconnect.

**Comment by Soham52** [comment]
ok, thanks
